config.js

```diff
@@ -19,6 +19,10 @@
     "featureManagerIsSSLEnabled": false,
     "featureManagerIsWAFEnabled": true,
     "homePageCards": ["ApplyDefaultSettingsCard", "SSLCard", "PurgeCacheCard"],
+    "moreSettingsCards": {
+        "container.moresettings.speed": ["CacheLevelCard", "AlwaysOnlineCard", "ImageOptimizationCard"],
+        "container.moresettings.security": ["SecurityLevelCard"]
+    },
```

Comment:
Does this mean `homePageCards` will have every single card since it's available? 

Or we can add a comment like 
```
     // Available Cards = ["SSLCard", "CacheLevelCard", .....]
     "homePageCards": ["ApplyDefaultSettingsCard", "SSLCard", "PurgeCacheCard"],
     "moreSettingsCards": {
         "container.moresettings.speed": ["CacheLevelCard", "AlwaysOnlineCard", "ImageOptimizationCard"],
         "container.moresettings.security": ["SecurityLevelCard"]
},
````